packages/grpc-js/src/make-client.ts

```diff
@@ -39,6 +39,7 @@ export interface MethodDefinition<RequestType, ResponseType> {
 }
 
 export interface ServiceDefinition {
+  // tslint:disable-next-line no-any
   [index: string]: MethodDefinition<any, any>;
```

Comment:
We could go with the following here, but I'm not sure if it wouldn't be a breaking change (although a safer choice).

```suggestion
  [index: string]: MethodDefinition<unknown, unknown>;
```

Reply:
It's hard for me to tell right now what the impact of changing this would be. I think it would make sense to defer this change to when we can take a broader look at the possibility of changing `any` to `unknown` across the codebase.

Reply:
Sure, let's leave it as-is then.